### ISSUE TITLE
Semantics

### ISSUE THREAD

**Issue by tenhobi** [leancodepl/patrol#1761]
Hi, what is the best way to work with Semantics labels? I tested Patrol with our app and while text, keys, ... worked, semantic labels did not.

From https://pub.dev/documentation/patrol_finders/latest/patrol_finders/createFinder.html it seems that text is converted to `CommonFinders.text` and not to `CommonFinders.bySemanticsLabel`.

Therefore

```dart
await $('Edit profile').tap(); // does not work
await $(K.editProfileButton).tap(); // works, but there is an issue with multiple keys in cases when you might have multiple items

await $(find.bySemanticsLabel('Edit profile')).tap(); // works, but not using Patrol finders
```

So... is using `flutter_test` API for Semantics what we are supposed to do? And could you mention the Semantics use case in docs in any way? Thanks.

**Comment by bartekpacia** [comment]
That's right, there's no equivalent to `CommonFinders.bySemanticsLabel` in Patrol. You have to use `flutter_test` APIs for this.

Feel free to contribute the info about Semantics to the docs.
